### PR TITLE
Sarah's edits to new core concepts and installation guides

### PR DESCRIPTION
**Pull request by sarah11918**
@FredKSchott - here are my proposed edits to your edits!  For your consideration. :)

Once we are happy with these files:
- CORE CONCEPTS: The following -new pages can be renamed and replace the existing pages, no changes to sidebar (yet):
-- Project Structure
-- Components
-- Pages
-- Layouts
(No changes yet to ...)
-- (Routing: we discussed moving "Routing" out of Core Concepts)
-- (Partial Hydration: to be discussed separately)

- SETUP Section updates:
-- Installation can be replaced with installation-new (renamed to remove -new)
-- manual-setup-new can be added to the sidebar (rename the file to remove -new and make sure Installation links to it properly)
-- Quick Start can be deleted and removed from the sidebar

### CODE REVIEW

File: src/pages/en/core-concepts/astro-components-new.md [hunk 1]
```diff
@@ -102,8 +115,11 @@ CSS `<style>` tags are also supported inside of the component template. They can
   /* scoped to the component, other H1s on the page remain the same */
   h1 { color: red }
 </style>
+
 <h1>Hello, world!</h1>
-```
+``` 
+
+> ⚠️ The styles defined here apply only to content written directly in the component's own component template. Children, and any imported components will **not** be styled by default. See our [Styling Guide](/en/guides/styling) for more information on applying styles.
```
Comment (FredKSchott) [inline src/pages/en/core-concepts/astro-components-new.md:122]: worth moving "See our [Styling Guide](/en/guides/styling) for more information on applying styles" out of this quote and into the top level to match the other "📚 ..." paragraphs on the page?

File: src/pages/en/installation-new.md [hunk 2]
```diff
@@ -27,16 +30,28 @@ pnpm create astro
...
 
-If `create-astro` starts successfully, you will see a list of [starter templates](https://github.com/withastro/astro/tree/main/examples) to choose from: 
+If `create-astro` starts successfully, you will see a short list of starter templates to choose from: 
 - `starter`: A great starter template for anyone wanting to explore Astro.
 - `minimal`: A template that just includes the bare minimium to get started.
 - `blog, portfolio, docs, etc`: opinionated themes for specific use-cases.
 
 If you choose the `starter` template, you will also be asked to select which [additional frameworks](/en/core-concepts/component-hydration) (React, Svelte, Vue, Solid, Preact), if any, you would like to include in your project. (Additional frameworks can also be added manually later.)
 
+> 💡 Or, you can install any of our [many starter templates](https://github.com/withastro/astro/tree/main/examples) directly via the command line:
```
Comment (FredKSchott) [inline src/pages/en/installation-new.md:40]: large quotes like this don't look great in our current stylesheet. Okay to move out of a quote?

**Review by FredKSchott — APPROVED** [review]
LGTM!!! left a couple of small comments but feel free to merge when you're ready.

I love that we're getting this consistency added! It's been so overdue

File: src/pages/en/core-concepts/astro-components-new.md (hunk 1 above)
Comment (sarah11918) [reply src/pages/en/core-concepts/astro-components-new.md:122]: Done.

File: src/pages/en/installation-new.md (hunk 2 above)
Comment (sarah11918) [reply src/pages/en/installation-new.md:40]: Done.

File: src/pages/en/core-concepts/astro-pages-new.md [hunk 3]
```diff
@@ -44,24 +44,24 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 </MySiteLayout>
 ```
 
-📚 Read more about [Layout components](/en/core-concepts/layouts-new) in Astro.
+📚 Read more about [layout components](/en/core-concepts/layouts-new) in Astro.
```
Comment (sarah11918) [inline src/pages/en/core-concepts/astro-pages-new.md:47]: Another -new link to be updated.

File: src/pages/en/guides/manual-setup-new.md [hunk 4]
```diff
@@ -99,4 +99,4 @@ If you have followed the steps above, your project directory should now look lik
 
 Congratulations, you're now set up to use Astro!
 
-If you'd like to continue a guided tour of Astro, read our main [Installation guide](/en/installation-new#3-start-). If you followed this guide completely, you can jump directly to [Step 3: Start](/en/installation-new#3-start-) to continue and learn how to run Astro for the first time.
\ No newline at end of file
+If you followed this guide completely, you can jump directly to [Step 3: Start](/en/installation-new#3-start-) to continue and learn how to run Astro for the first time.
```
Comment (sarah11918) [inline src/pages/en/guides/manual-setup-new.md:102]: Another -new link to be updated

**Pull request merged**